test/integration/preflight/preflight_test.go

```diff
@@ -70,8 +104,8 @@ func (p PlaybookTest) checkExitCode(t *testing.T, got, want int, cmd *exec.Cmd,
 	if got == want {
 		return
 	}
-	t.Fatalf("got exit code %v, want %v", got, want)
 	p.logCmdAndOutput(t, cmd, output)
+	t.Fatalf("got exit code %v, wanted %v", got, want)
```

Comment:
> that may be, but is it too much to ask for the two verbs to be in the same tense? :)

Yeah, I don't know why they use that that way.

My reasoning is "I (still) want XYZ". With "wanted" having maybe a slightly less implicative semantics (I might still want, or I might not want it any more). But that may sound like paranoia :P

My understanding of the "Go-way" is that a lot of times we go with shorter / more concise, even though in Python I speak a completely different dialect, preferring explicit / longer names.